src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml

```diff
@@ -14,38 +14,51 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
-  <domain name="CHIP" />
-
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/data_model/Group-Key-Management-Cluster.adoc
+Parameters: zap attribute=in-progress ignore-errored provisional-policy=loose sdkRoot=./connectedhomeip/ specRoot=./connectedhomeip-spec/ ./connectedhomeip-spec/src/data_model/Group-Key-Management-Cluster.adoc 
+Git: 1.5-base-19-g551278a1a
+Alchemy: v1.5.56
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="CHIP"/>
   <struct name="GroupKeyMapStruct" isFabricScoped="true">
     <cluster code="0x003F"/>
     <item fieldId="1" name="GroupId" type="group_id"/>
-    <item fieldId="2" name="GroupKeySetID" type="int16u"/>
+    <item fieldId="2" name="GroupKeySetID" type="int16u" min="1"/>
   </struct>
 
   <struct name="GroupInfoMapStruct" isFabricScoped="true">
     <cluster code="0x003F"/>
     <item fieldId="1" name="GroupId" type="group_id"/>
-    <item fieldId="2" name="Endpoints" type="endpoint_no" array="true"/>
+    <item fieldId="2" name="Endpoints" type="endpoint_no" array="true" minLength="1"/>
     <item fieldId="3" name="GroupName" type="char_string" length="16" optional="true"/>
   </struct>
 
+  <enum name="GroupKeyMulticastPolicyEnum" type="enum8">
+    <cluster code="0x003F"/>
+    <item name="PerGroupID" value="0x00"/>
+    <item name="AllNodes" value="0x01"/>
+  </enum>
+
   <enum name="GroupKeySecurityPolicyEnum" type="enum8">
     <cluster code="0x003F"/>
     <item name="TrustFirst" value="0x00"/>
     <item name="CacheAndSync" value="0x01"/>
   </enum>
 
-  <struct name="GroupKeySetStruct">
+  <struct name="GroupKeySetStruct" isFabricScoped="true">
```

Comment:
This is a spec bug, which was not correctly fixed in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11696 as far as I can tell.  As in, this struct should not be fabric-scoped in the spec.